File: Jointdistr/bivnorm.jl
using Plots, LinearAlgebra, Random
N = 1000; rho = 0.8;
Sigma = [1 rho; rho 1];
B = cholesky(Sigma).L;
# lower-triangular Cholesky matrix
x = B*randn(2,N);
scatter(x[1,:],x[2,:],ms=2,msw=0,legend=false)

File: Jointdistr/bivnorm.r
N= 1000;
rho = 0.8;
Sigma<-rbind(c(1,rho), c(rho,1));
B=t(chol(Sigma))
x=B%*%matrix(rnorm(N),2)
plot(x[1,],x[2,],pch=18,col="blue")